tests/Sulu/Component/Webspace/Loader/XmlFileLoaderTest.php

```diff
@@ -306,17 +306,6 @@ public function testLoadWithNotExistingDefault()
         );
     }
 
-    public function testLoadWithoutDefaultLocalization()
```

Comment:
i have changed this. if no default is selected (with xml) first is default. therfor this testcase will fail and i have removed it!